sql/addons_view_v1.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
+  @submission_date as submission_date,
+  document_id,
+  client_id,
+  sample_id,
+  subsession_start_date,
+  normalized_channel,
+  addons_array.element.*
```

Comment:
`addons_array` isn't an array, and [should use `_` prefix for a generated column name not meant for output](https://github.com/mozilla/bigquery-etl/blob/master/README.md#queries), so I think this should be named `_addon`: 

```suggestion
  _addon.element.*
```